eqt/threading/QtThreading.py

```diff
@@ -40,15 +38,16 @@ def __init__(self, fn, *args, **kwargs):
     def run(self):
         """
         Run the worker. Emits signals based on run state.
-        Signals:
-            - Error: Emitted when an exception is thrown in the workers function.
-            - Result: Emitted if function completes successfully. Contains the return value of the function.
-            - Finished: Emitted on completion of the worker thread.
 
+        Signals
+        -------
+        - Error: An exception is thrown in the workers function.
+        - Result: Contains the return value of a function that has just completed successfully.
+        - Finished: Worker thread has completed.
         """
         try:
             result = self.fn(*self.args, **self.kwargs)
-        except:
+        except Exception: # TODO: check
```

Comment:
I'm not sure whether this is intended to be `Exception` or (as before) really catch everything - including `KeyboardInterrupt`s etc, i.e.:

```suggestion
        except BaseException: # TODO: check
```